Razorpay.php

```diff
@@ -1,10 +1,13 @@
 <?php
 
-// Include Requests
-require_once __DIR__.'/libs/Requests-1.6.1/library/Requests.php';
+// Include Requests only if not already defined
+if (class_exists('Requests') === false)
+{
+    require_once __DIR__.'/libs/Requests-1.6.1/library/Requests.php';
 
-// Register requests autoloader
-Requests::register_autoloader();
+    // Register requests autoloader
+    Requests::register_autoloader();
+}
 
```

Comment:
@shashankkumar But, if `Requests` is already defined, (like in the case of wordpress), we shouldn't throw an exception, right? Instead, the purpose of this check is to ensure we are not loading a class that is already existing within the framework, and in that case, throwing an exception would not be the correct behaviour, right?